src/audio/audio.c

```diff
@@ -1714,7 +1714,8 @@ void audio_init1()
 			}
 		}
 		m++;
-	} while(m + 1 < 0x2E);
+	// 46 is the number of files in ride_music_info_list
+	} while(m + 1 < 46);
```

Comment:
I'll refactor it to a for loop.
With 'macro' do you mean countof? It doesn't seem to work.